JoonasHeinonen/escape-from-zordoom-prison
# hot_fix:arena_sniper_rifle

## [hot_fix:arena_sniper_rifle](https://trello.com/b/vThoQnyu/escape-from-zordoom-prison)

<Paste the media recorded of your feature in the action, e.g. a video or an image>
Added a new level to the game where you can fight people and make money. As well as fixed a bug where the checkpoint system is now working
## How to test:
 -pull down this branch.
-select the arena level
 -spawn in and talk to the shark man
- talk to him and get to the menu
- click on the first fight and the player will spawn in
- fight the nef head bad guy
- when they die the player will spawn back in next to the shark man and the process will start over again.
- the player should get 100 bolts for completing this fight.
- Check the check point system by:
- select the test level
- go to the check point then die 
- you should spawn back in the checkpoint 


## Sources cited:
 - youtube, big help from the godot discord server!
